techniques/failures/F91.html

```diff
@@ -133,32 +130,32 @@ <h3>Example 1: Headers not marked up appropriately</h3>
   &lt;/tr&gt;  
 &lt;/table&gt;</code></pre>
             <p class="working-example">
-               <a href="../../working-examples/failure-td-not-th/">View example 1 (opens in same browser window)</a>
+               <a href="https://www.w3.org/WAI/WCAG22/working-examples/failure-td-not-th/">例 1 の実装を見る (同じブラウザウィンドウで開く)</a>
 							     </p>
          
       </section>
    </section>
 
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
      <li><a href="../html/H43">H43: Using id and headers attributes to associate data cells with header cells in data tables</a></li> 
      <li><a href="../html/H51">H51: Using table markup to present tabular information</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
-         <p>For all data tables, check if table headers can be correctly programmatically determined by use of one of the following mechanisms: 
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
+         <p>全てのデータテーブルについて、次のいずれかのメカニズムを用いて、テーブル見出しが正しくプログラムによる解釈が可能かどうかを確認する: 
 						</p>
          <ol>
-            <li>headers marked up with table header (<code class="language-html">th</code>) elements</li>
-         	<li><code class="language-html">scope</code> attributes on <code class="language-html">th</code> for tables with more than a single row or column of table headers.</li>
-            <li>headers and data cells associated using <code class="language-html">headers</code> and <code class="language-html">id</code> attributes</li>
-            <li>headers marked up as <code class="language-html">td</code> elements with the <code class="language-html">scope</code> attribute</li>
-            <li>headers marked up with ARIA role attributes <code class="language-html">rowheader</code> or <code class="language-html">columnheader</code></li>
+            <li>テーブル見出し要素 (<code class="language-html">th</code>) でマークアップされた見出し</li>
+         	<li>テーブル見出しが 1 行より多い又は 1 列より多いテーブルの  on <code class="language-html">th</code> における <code class="language-html">scope</code> 属性</li>
```

Comment:
原文の記述 (on) が残っているのでトルツメしました。

```suggestion
         	<li>テーブル見出しが 1 行より多い又は 1 列より多いテーブルの  <code class="language-html">th</code> における <code class="language-html">scope</code> 属性</li>
```

Reply:
ありがとうございます。…全角スペースになってますね。
```suggestion
         	<li>テーブル見出しが 1 行より多い又は 1 列より多いテーブルの <code class="language-html">th</code> における <code class="language-html">scope</code> 属性</li>
```